helpers/utils.ts

```diff
@@ -182,3 +183,22 @@ export function asPercentageValue(value: BigNumber.Value, base: BigNumber.Value)
     asDecimal: val.div(base),
   }
 }
+
+export async function getLatestBlock(provider: JsonRpcProvider) {
+  return provider.getBlockNumber()
+}
+
+export async function getRecentBlock({
+  provider,
+  offset,
+  roundToNearest,
+}: {
+  provider: JsonRpcProvider
+  offset: number
+  roundToNearest?: number
```

Comment:
I think you can add a default value in parameter definitions.

Reply:
done